iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender.inl

```diff
@@ -45,12 +45,11 @@ ChunkSender<ChunkSenderDataType>::tryAllocate(const uint32_t payloadSize, const
     const uint32_t neededChunkSize = getMembers()->m_memoryMgr->sizeWithChunkHeaderStruct(payloadSize);
```

Comment:
with `sizeWithChunkHeaderStruct` we must be careful to also consider the optional CustomChunkHeader

Reply:
yes, that's part of the next PR